packages/wdio-runner/src/utils.ts

```diff
@@ -64,7 +65,15 @@ export async function initialiseInstance (
     if (config.sessionId) {
         log.debug(`attach to session with id ${config.sessionId}`)
         config.capabilities = sanitizeCaps(capabilities)
-        return attach({ ...config } as Required<ConfigWithSessionId>)
+
+        /**
+         * propagate connection details defined by services or user in capabilities
+         */
+        const { protocol, hostname, port, path } = pick(
```

Comment:
lodash.pick seems to do the same as destructuring, can you please verify?

Reply:
this is using `lodash.pick`, see top of file:

```ts
import pick from 'lodash.pick'
```

Reply:
ahh I understand now, I know there was a reason, let me double check

Reply:
you are right, lodash is not needed here, good catch